src/server.ts

```diff
@@ -206,16 +206,17 @@ const ${hashId(chunk.id)} = ${chunk.code}
 
   const dynamicImportCode = `
 const __vite_import_cache__ = Object.create({})
-async function __vite_ssr_import__ (id) {
-  if (__vite_import_cache__[id]) {
-    return __vite_import_cache__[id]
-  }
-  const mod = await $chunks[id]()
-  if (mod && !('default' in mod)) {
-    mod.default = mod
+async function __vite_ssr_import__(id) {
```

Comment:
I guess we no longer need async keyword
```suggestion
function __vite_ssr_import__(id) {
```